python/pyproject.toml

```diff
@@ -3,26 +3,28 @@ name = "ccao"
 version = "1.3.0"
 description = "Convenience Functions and Datasets for the Cook County Assessor's Office"
 readme = "README.md"
-requires-python = ">=3.10"
+requires-python = ">=3.9"
 authors = [
     {name = "Jean Cochrane", email="jean.cochrane@cookcountyil.gov"},
     {name = "Dan Snow", email="daniel.snow@cookcountyil.gov"},
 ]
 dependencies = [
-    "pandas>=2.2.3",
+    "pandas>=1.4.0,<=2.3.0",
+    "numpy>=1.23.0,<=2.2.0"
```

Comment:
The upper bounds are only present here because we're not testing anything higher in the `pytest-coverage` workflow, given the weird matrix in the workflow. If we can think of a way to remove the hardcoded upper bounds from the workflow matrix, we could remove the upper bound here. We could also just remove the upper bound from this config and accept that we're potentially not testing the latest versions in the `pytest-coverage` workflow.